src/components/DataTable/components/SbDataTableHeaderCell.vue

```diff
@@ -74,6 +74,20 @@ export default {
     hasHeaderSlot() {
       return this.column?.scopedSlots?.header
     },
+
+    computedClasses() {
+      let classesList = []
```

Comment:
how about  changing it to this, it seems more readable

```
return {
'sb-data-table--show-icon':  this.showSortIcon,
'sb-data-table__head-cell--centered': this.column?.isContentCentered,
'sb-data-table__head-cell--sortable': (this.column?.sortable
}

Reply:
Absolutely, I'll add this change thanks 👍